code/_global_vars/lists/flavor.dm

```diff
@@ -0,0 +1,17 @@
+GLOBAL_LIST_INIT(greek_letters, list("Alpha", "Beta", "Gamma", "Delta",
+	"Epsilon", "Zeta", "Eta", "Theta", "Iota", "Kappa", "Lambda", "Mu",
+	"Nu", "Xi", "Omicron", "Pi", "Rho", "Sigma", "Tau", "Upsilon", "Phi",
+	"Chi", "Psi", "Omega"))
+
+GLOBAL_LIST_INIT(phonetic_alphabet, list("Alpha", "Bravo", "Charlie",
+	"Delta", "Echo", "Foxtrot", "Golf", "Hotel", "India", "Juliet",
+	"Kilo", "Lima", "Mike", "November", "Oscar", "Papa", "Quebec",
+	"Romeo", "Sierra", "Tango", "Uniform", "Victor", "Whiskey", "X-ray",
+	"Yankee", "Zulu"))
+
+// Used by robots and robot preferences.
+GLOBAL_LIST_INIT(robot_module_types, list(
+	"Standard", "Engineering", "Rescue",  "Medical",
+	"Miner",    "Janitor",     "Service",  "Clerical", "Security",
+	"Research" , "Construction"
+))
```

Comment:
That list should be build automaticly based on robot modules list.
Right now that list unsynced with real robot modules list.
Check `modules/mob/living/silicon/robot/robot_modules.dm`